From 29325d80ab75d3cccc329b15a6137dead5bb7b1a Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 24 Nov 2016 10:43:42 -0800
Subject: [PATCH] TOOLS-1610 sdcadm's appending "-$buildstamp" to the version
 was naive and broke 'sdcadm self-update --latest' (revert TOOLS-1599)

---
 CHANGES.md    | 6 ++++++
 package.json  | 2 +-
 tools/mk-shar | 2 +-
 3 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index b1da15d..08db4f1 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,12 @@
 
 # sdcadm Changelog
 
+## 1.14.0
+
+- TOOLS-1610 Drop the "-$buildstamp" suffix on the sdcadm image version field
+  that was added by TOOLS-1599, because it breaks 'sdcadm self-update
+  --latest'.
+
 ## 1.13.0
 
 - TOOLS-1610 Bump ver as a workaround to get 'sdcadm self-update --latest'
diff --git a/package.json b/package.json
index d7bacbf..915828a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a SmartDataCenter (SDC) standup",
-  "version": "1.13.0",
+  "version": "1.14.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
diff --git a/tools/mk-shar b/tools/mk-shar
index fb4e2e8..97fa44e 100755
--- a/tools/mk-shar
+++ b/tools/mk-shar
@@ -177,7 +177,7 @@ cat <<EOM >$OUTMANIFEST
     "uuid": "$(uuid)",
     "owner": "00000000-0000-0000-0000-000000000000",
     "name": "sdcadm",
-    "version": "$VERSION-$STAMP",
+    "version": "$VERSION",
     "description": "Administer SmartDataCenter standups",
     "disabled": false,
     "public": false,
-- 
2.21.0

